USAGE.md

```diff
@@ -146,7 +146,7 @@ aws lambda invoke \
 ## Dataset Version Endpoint Usage Examples
 
 - Set Dataset Space Endpoint Lambda function name
-  
+
```

Comment:
I believe these newlines will break some Markdown renderers. If the original Markdown isn't rendering properly somewhere, please let me know. Also, all the whitespace changes in this commit probably belongs in a different PR.

Reply:
They must have been done by some automatic formatting in my editor. I'll remove them.

Reply:
Actually, it is not new lines but removed two spaces at the beginning of new lines.